actions/bcr-pr-review-notifier/index.js

```diff
@@ -84,7 +84,16 @@ async function notifyMaintainers(octokit, owner, repo, prNumber, maintainersMap)
   }
 
   // Notify maintainers based on grouped module lists
+  prAuthor = context.payload.pull_request.user.login;
   for (const [modulesList, maintainers] of moduleListToMaintainers.entries()) {
+    // Skip notifying the PR author if they are one of the module maintainers
+    if (maintainers.has(`@${prAuthor}`)) {
+      console.log(`Skipping notifying PR author ${prAuthor} from the maintainers list for modules: ${modulesList}`);
+      maintainers.delete(`@${prAuthor}`);
```

Comment:
I have no idea about JS, but does this work given the const? And will it mutate the original list or does JS create a copy?

Reply:
The `const` shouldn't matter, tested at https://github.com/meteorcloudy/bazel-central-registry/actions/runs/7785697256/job/21228879231?pr=11

However, indeed it's better to not modify the original maintainers list.

Reply:
Thanks for the clarification!